service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ObservationStatementMapper.java

```diff
@@ -161,23 +166,29 @@ private String prepareComment(Observation observation) {
 
                 Optional.ofNullable(coding).ifPresent(code -> {
                     if (code.hasCode() || code.hasDisplay()) {
-                        commentBuilder.append("Interpretation Code: ");
+                        commentBuilder.append(INTERPRETATION_CODE_PREFIX);
                     }
                     if (code.hasCode()) {
                         commentBuilder.append(coding.getCode()).append(StringUtils.SPACE);
                     }
                     if (code.hasDisplay()) {
                         commentBuilder.append(code.getDisplay()).append(StringUtils.SPACE);
                     }
-
-                    if (interpretation.hasText()) {
-                        commentBuilder.append("Interpretation Text: ").append(interpretation.getText()).append(StringUtils.SPACE);
-                    }
                 });
             }
         }
 
-        commentBuilder.append(observation.hasComment() ? observation.getComment() : StringUtils.EMPTY);
+        Optional<String> interpretationText = StringUtils.isNotBlank(observation.getInterpretation().getText())
+            ? Optional.of(observation.getInterpretation().getText())
+            : Optional.empty();
+        String interpretationTextAndComment = Stream.concat(
+            interpretationText.stream(),
+            Optional.ofNullable(observation.getComment()).stream()
```

Comment:
Should we do the same with comment as we do with interpretation, i.e. use `StringUtils.isNotBlank`?